flight/PiOS/STM32F4xx/inc/pios_frsky_rssi.h

```diff
@@ -0,0 +1,46 @@
+/**
+ ******************************************************************************
+ * @addtogroup PIOS PIOS Core hardware abstraction layer
+ * @{
+ * @addtogroup PIOS driver for PWM RSSI input
+ * @brief Input RSSI PWM
+ * @{
+ *
+ * @file       pios_frsky_rssi.h
+ * @author     Tau Labs, http://taulabs.org, Copyright (C) 2013-2015.
```

Comment:
Copyright just 2015, right?

This file should probably go in pios_common. I don't think there is a problem having the ```TIM_ICInitTypeDef``` in there is other pios driver include files have it there.

Also strictly to follow our normal conveniton this should be two files. a _priv.h variant that has the cfg/Init method and then the public API component that has the _Get() method.